aichatgirls.py

```diff
@@ -18,8 +18,9 @@
 from chatCommand import chat_command
 from datetime import datetime
 import discord
+import encryption
 from generate import generate_prompt_response
-from loadCharacterCard import load_character_card
+import loadCharacterCard

```

Comment:
Now you have to write `loadCharacterCard.load_character_card` instead of just `load_character_card`. You might just be making things harder here!